templates/docs/patterns/equal-height-row.md

```diff
@@ -0,0 +1,99 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Equal height row | Components
+---
+
+The equal height row component aims to provide consistent alignment for grid items within a row format. This is achieved using the CSS `subgrid` feature which allows column grids to share the same layout used in the parent row grid.
+
+This component is an extension of the [grid component](/docs/patterns/grid). There are two ways to use this component:
+
+1. Used as a four column row grid (`.p-equal-height-row`). Each column (`.p-equal-height-row__col`) within the row is a css grid, and may have up to four grid items (`.p-equal-height-row__item`).
```

Comment:
```suggestion
1. Used as a four column row grid (`.p-equal-height-row`) spanning the whole width of the default page grid. Each column (`.p-equal-height-row__col`) within the component is a sub-grid may have up to four items (`.p-equal-height-row__item`).
```

Reply:
Actually, I think this description can be moved inside the "Four column row" section, above the example.